lib/jekyll-relative-links/generator.rb

```diff
@@ -65,7 +65,9 @@ def url_for_path(path)
       extension = File.extname(path)
       return unless markdown_extension?(extension)
 
-      page = site.pages.find { |p| p.path == path }
+      page = site.pages.find { |p| p.path == path } ||
+        site.docs_to_write.find { |p| p.relative_path == path } ||
+        site.static_files.find { |p| p.relative_path == path }
```

Comment:
If we're expanding this to support static files, we'll want to add a test as well.